project/BlenderWang9487/README.rst

```diff
@@ -0,0 +1,106 @@
+========================
+DummyML
+========================
+
+
+
+Basic Information
+=================
+
+A dummy ML library for NSD course term project.
+
+Here is my Github repository:
+`DummyML <https://github.com/BlenderWang9487/DummyML.git>`_
+
+Problem to Solve
+================
+
+I want to build a machine learning library based on some mathematical theories.
+
+There are many easy-to-use hybrid frameworks that built for deep learning, 
+but fewer for other traditional ML algorithms. So I'm going to build one.
+
+Some regression, classification and clustering models (like Gaussian linear 
```

Comment:
I like the goals of addressing the basics.  We can learn a lot of coding skills by implementing well-knows algorithms.